### PR TITLE
Bind hb_face_reference_table APIs

### PR DESCRIPTION
**Pull request by mcratt**
face.get_table_blob(tag) or face.table_blob(tag) ?

### CODE REVIEW

File: src/uharfbuzz/_harfbuzz.pyx [hunk 1]
```diff
@@ -635,6 +635,14 @@ cdef class Face:
             raise MemoryError()
         return Blob.from_ptr(blob)
 
+    def get_table_blob(self, tag: str) -> Blob:
```
Comment (khaledhosny) [inline src/uharfbuzz/_harfbuzz.pyx:638]:
```suggestion
    def reference_table(self, tag: str) -> Blob:
```

Comment (mcratt) [reply src/uharfbuzz/_harfbuzz.pyx]: ok

File: tests/test_uharfbuzz.py [hunk 2]
```diff
@@ -786,6 +786,177 @@ def test_list_names_with_user_names(self, mutatorsans):
             "MutatorMathTest-Medium_Narrow_I",
         ]
 
+    def test_reference_table(self, blankfont):
```
Comment (khaledhosny) [inline tests/test_uharfbuzz.py:789]: I don’t think we need this overly detailed test. It should be enough to check for the table length, and may be the binary data. No need to try to decode it.

Comment (mcratt) [reply tests/test_uharfbuzz.py:789]: ok

File: tests/test_uharfbuzz.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import sys
 import platform
 import pytest
-
+import struct
```
Comment (khaledhosny) [inline tests/test_uharfbuzz.py:6]: unused import

Comment (mcratt) [reply tests/test_uharfbuzz.py]: ok

**Comment by khaledhosny** [comment]
Thanks!

**Pull request merged**